mission/config/subconfigs/vehicle_lock_info.hpp

```diff
@@ -46,221 +124,249 @@ class vn_b_armor_m113_acav_04 : acav {};
 class vn_b_armor_m113_acav_05 : acav {};
 class vn_b_armor_m113_acav_06 : acav {};
 
+// M113 command and ambulance variants
+class vn_b_armor_m577_01 : acav {};
+class vn_b_armor_m577_02 : acav {};
+
+// M113 mortar and flamethrower
+class vn_b_armor_m125_01 : armor {};
+class vn_b_armor_m132_01 : armor {};
+
+//M41 Walker bulldog
+class vn_b_armor_m41_01_01 : armor {};
+
+//Type 63 tank
+class vn_i_armor_type63_01 : arvn {};
+
+// M48 Patton
+class vn_b_armor_m48_01_01 : armor {};
+
+// M67 flame tank
+class vn_b_armor_m67_01_01 : armor {};
+
```

Comment:
no locking the USMC variants to USMC? `vn_b_armor_m48_01_02` and `vn_b_armor_m67_01_02`

Reply:
resolved